screens/AppNavigator/screens/Dex/DexScreen.tsx

```diff
@@ -35,6 +38,18 @@ export function DexScreen (): JSX.Element {
     return wallet.tokens.filter(({ isLPS }) => isLPS).map(data => ({ type: 'your', data: data }))
   })
 
+  const onAdd = (data: PoolPairData): void => {
+    navigation.navigate('AddLiquidity', { pair: data })
+  }
+
+  const onRemove = (): void => {
```

Comment:
you can remove the unused methods first

Reply:
They are in use, attached to the button, just have to plug in the navigation here when the destination page is ready.